Engines/Wine/Plugins/override DLL/script.js

```diff
@@ -3,23 +3,21 @@ const Regedit = include("engines.wine.plugins.regedit");
 module.default = class OverrideDLL {
     constructor(wine) {
         this.wine = wine;
-        this.modes = {};
+        this.modes = new Map();
     }
 
     withMode(mode, libraries) {
-        this.modes[mode] = libraries;
+        this.modes.set(mode, libraries);
 
         return this;
     }
 
     go() {
         let regeditFileContent = `REGEDIT4\n\n[HKEY_CURRENT_USER\\Software\\Wine\\DllOverrides]\n`;
 
-        Object.entries(this.modes)
-            .map(([mode, libraries]) => libraries.map(library => [library, mode]))
```

Comment:
Just as a sidenot, I think the following change would be enough to fix the issue but I agree that your code is more readable:

```javascript
.flatMap(([mode, libraries]) => libraries.map(library => [library, mode]))
```